ddl/db_integration_test.go

```diff
@@ -1945,3 +1949,131 @@ func (s *testIntegrationSuite3) TestParserIssue284(c *C) {
 	tk.MustExec("drop table test.t_parser_issue_284")
 	tk.MustExec("drop table test.t_parser_issue_284_2")
 }
+
+func (s *testIntegrationSerialSuite) TestAutoRandom(c *C) {
```

Comment:
Can we put this test into `serial_test.go` which contains all the DDL serial tests?